app/code/core/Mage/Page/Helper/Data.php

```diff
@@ -19,5 +19,11 @@
  */
 class Mage_Page_Helper_Data extends Mage_Core_Helper_Abstract
 {
+    public const XML_PATH_DEMO_NOTICE = 'design/head/demonotice';
+
+    public const XML_PATH_DEMO_NOTICE_TEXT = 'design/head/demonotice_text';
```

Comment:
Drop the word "DEMO"?
```suggestion
    public const XML_PATH_STORE_NOTICE_ENABLED = 'design/head/demonotice';

    public const XML_PATH_STORE_NOTICE_TEXT = 'design/head/store_notice_text';
```